packages/ethereum-contracts/foundry.toml

```diff
@@ -2,7 +2,7 @@
 root = '../..'
 src = 'packages/ethereum-contracts/contracts'
 test = 'packages/ethereum-contracts/test/foundry/'
-solc_version = "0.8.19"
+solc_version = "0.8.21"
```

Comment:
specifying solc_version will restrict foundry from downloading it itself, a interesting feature and good for our immutable CI.

Ref: https://book.getfoundry.sh/reference/config/solidity-compiler#solc_version